docs/index.md

```diff
@@ -127,28 +127,3 @@ glossary
 {doc}`teachers-training/index`
 
 : How To Give Technical Trainings
```

Comment:
If we remove these toctree entries, and we will remove the files eventually, why not remove the files now?

Reply:
Some are referenced, like code (info about checking out branches of the training code).
Some should be removed. This could be done while working on content next days. > @pbauer 
teachers training: While adding authors instructions to markdown, I'll add a reference in second toc in main navigation.